opencti-platform/opencti-graphql/tests/02-integration/01-database/stixCyberObservable-domain-test.ts

```diff
@@ -0,0 +1,92 @@
+import { describe, it, expect } from 'vitest';
+import { Readable } from 'stream';
+import { addStixCyberObservable, stixCyberObservableDelete } from '../../../src/domain/stixCyberObservable';
+import { ADMIN_USER, testContext } from '../../utils/testQuery';
+import type { AuthContext } from '../../../src/types/user';
+import { mergeEntities, storeLoadByIdWithRefs } from '../../../src/database/middleware';
+import type { BasicStoreCommon } from '../../../src/types/store';
+import { requestFileFromStorageAsAdmin } from '../../utils/testQueryHelper';
+import { paginatedForPathWithEnrichment } from '../../../src/modules/internal/document/document-domain';
+import { logApp } from '../../../src/config/conf';
+
+describe('Testing Artifact merge with files on S3 (see issue 6258)', () => {
```

Comment:
I don't think we should put the issue number in the test description (that ends up in test reports and so on). More suited in a simple comment IMO.